ocs_ci/ocs/ui/views.py

```
@@ -3370,6 +3370,28 @@
         "//span[text()='Secret name']/ancestor::div[contains(@class, 'form__group')]//span[@class='odf-resource-item']",
         By.XPATH,
     ),
+    # Object row actions (object browser)
+    "object_row_kebab": (
+        "//tr[.//td[@data-label='Name' and contains(normalize-space(),'{}')]]"
+        "//button[@aria-label='Kebab toggle']",
+        By.XPATH,
+    ),
+    "object_action_preview": (
+        "//button[contains(@class,'c-menu__item')" " and normalize-space()='Preview']",
```

Ruff (0.15.21)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)